dashboard/src/app/page.tsx

```diff
@@ -0,0 +1,17 @@
+import { redirect } from 'next/navigation'
+import getWhoAmI from './getWhoami'
+
+export default async function Home() {
+  const whoAmI = await getWhoAmI()
```

Comment:
I was thinking why not do this in middleware but yes Next.js doesn't support node apis in `middleware.ts`